### PR TITLE
Add roslyn bump instructions next to roslyn-binaries

### PR DESCRIPTION
**Pull request by kg**
@monojenkins skip

### CODE REVIEW

File: external/how-to-bump-roslyn-binaries.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+12. Once you commit the submodule bump to your local repository, you can do test builds. **If you do test builds without committing the bump, the build process will erase the bump.**
+13. Now we need to change the roslyn paths used by the build process - we're still using the older roslyn folder. We do this by changing the CSC_LOCATION and VBCS_LOCATION variables in configure.ac. They should be something like ```Microsoft.Net.Compilers/3.1.0/csc.exe```, change the version number in those paths, save, and make a local commit. Then do the full autogen, make, make install routine and do some basic tests. This should work.
+14. Next, make sure the bump didn't break csi. If you managed the .rsp files correctly in the last step, it should work. This has test coverage, so just pay attention to the automated tests for it.
+15. Ensure the 'Linux x64 - prefix sanity checks' lane passed on CI, even if it's currently disabled or not required. This is required for a bump. If it isn't visible on the PR statuses, manually trigger the lane using Jenkins.
```
Comment (jaykrell) [inline external/how-to-bump-roslyn-binaries.md]: "using Jenkins" do you mean @monojenkins something? Be more specific?

Comment (kg) [reply external/how-to-bump-roslyn-binaries.md]: By going to jenkins and triggering the lane. It's not monojenkins. Do you think I need to add a URL or something? I'm not sure we can link to that

Comment (jaykrell) [reply external/how-to-bump-roslyn-binaries.md]: Agreed about the uncertainty then. I don't know.
33 steps, ugh.

Comment (kg) [reply external/how-to-bump-roslyn-binaries.md]: I'm sure it will turn out that I missed at least one :)

**Pull request merged**